scijava/scijava-ops/src/main/java/org/scijava/struct/Member.java

```diff
@@ -16,6 +16,9 @@
 	/** Unique name of the member. */
 	String getKey();
 
+	/** Description of the member. */
+	String getDescription();
```

Comment:
I think I'd prefer `default String getDescription() { return ""; }` instead of abstract, here. The description is a thing that some struct implementations may not want to deal with. The original idea of the struct interfaces was to keep them kind of thin... while I do agree we need `getDescription()` to be a method here, but the default impl would give us the best of both worlds.

Reply:
Fixed in  67ef055